lxc/utils.go

```diff
@@ -39,6 +39,34 @@ func runBatch(names []string, action func(name string) error) []batchResult {
 	return results
 }
 
+// Get devices from a list of profiles.
+func getProfileDevices(destRemote lxd.InstanceServer, serverSideProfiles []string) (map[string]map[string]string, error) {
```

Comment:
We usually start the description with the functions name like so: `// getProfileDevices returns devices from a list of profiles.` 
I guess this file is a bad example as the other functions don't do it so it's not a blocker.